### ISSUE TITLE
QUESTION: topnav_right duplicates menu item 

### ISSUE THREAD

**Issue by Nardusgc** [jeroennoten/Laravel-AdminLTE#543]
Hi, 

When using topnav_right attribute 
```[
                        'text' => 'Login',
                        'url' => 'login',
                        'icon' => 'fas fa-fw fa-user',
                        'topnav_right' => 'true',
                    ]
```

it duplicated the menu item on the left and right? I'm i missing something? 
![Selection_003](https://user-images.githubusercontent.com/3862600/79867338-807ee680-83de-11ea-93bb-0ce650b83c0a.png)

**Comment by lacodimizer** [comment]
Thats correct, if you have set the config "layout_topnav" to true.

**Comment by Nardusgc** [comment]
@lacodimizer thanks for the quick reply. How do i then get the login item to only display at the top right of the nav and not on the left aswell?

**Comment by lacodimizer** [comment]
Thats a good question, my idea is, that a menu item will be not set to topnav_left, if it set to topnav_right. It makes not sense, to set a menu in both navigations...

**Comment by REJack** [comment]
I forgot to modify the `if` here:
https://github.com/jeroennoten/Laravel-AdminLTE/blob/eb9e1399765272d008d1b8b02f201cc4c7b32328/resources/views/partials/menu-item-top-nav-left.blade.php#L1

It should look like this (not tested):
```php
 @if((config('adminlte.layout_topnav') && (isset($item['topnav_right']) && !$item['topnav_right'])) or (isset($item['topnav']) && $item['topnav']))) 
```

**Comment by lacodimizer** [comment]
@Nardusgc @REJack fix comes in #545
